apps/condo/domains/notification/transports/push.js

```diff
@@ -16,15 +16,17 @@ const {
     PUSH_TRANSPORT_HUAWEI,
 } = require('@condo/domains/notification/constants/constants')
 const { renderTemplate } = require('@condo/domains/notification/templates')
-const { RemoteClient } = require('@condo/domains/notification/utils/serverSchema')
-
-const logger = getLogger('messaging/sendNotification')
 
 const ADAPTERS = {
     [PUSH_TRANSPORT_FIREBASE]: new FirebaseAdapter(),
     [PUSH_TRANSPORT_HUAWEI]: new HCMAdapter(),
     [PUSH_TRANSPORT_APPLE]: new AppleAdapter(),
 }
+const responseResolversByTransport = {
+    [PUSH_TRANSPORT_FIREBASE]: async (result, isVoIP) => await responseResolver(result, isVoIP),
+    [PUSH_TRANSPORT_HUAWEI]: () => { return 'not implemented'},
+    [PUSH_TRANSPORT_APPLE]: () => { return 'not implemented'},
```

Comment:
https://stackoverflow.com/questions/19782470/what-happen-if-a-apns-device-token-expired
https://stackoverflow.com/questions/6652242/does-the-apns-device-token-ever-change-once-created